src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -113,15 +119,33 @@ export class DomEventPlugin implements Plugin {
         return '';
     }
 
+    private getElementCSSLocator(event: Event, domEvent: TargetDomEvent) {
+        if (!event.target) {
+            return '';
+        }
+
+        if (event.target instanceof Element && domEvent.cssLocator) {
+            return domEvent.cssLocator;
+        }
+
+        if (event.target instanceof Node) {
+            return event.target.nodeName;
+        }
+    }
```

Comment:
Unlike the `elementId`, we can consider the CSS locator optional. In this case we can return a CSS locator if one exists, or `undefined` otherwise.
```javascript
private getElementCSSLocator(domEvent: TargetDomEvent) {
    if (domEvent.cssLocator) {
        return domEvent.cssLocator;
    }
    return undefined;
}
```

Reply:
Refactored as suggested.